janus_core/cli/geomopt.py

```diff
@@ -33,6 +33,61 @@
 app = Typer()
 
 
+def _set_minimize_kwargs(
+    minimize_kwargs: dict[str, Any],
+    traj: Optional[str],
+    vectors_only: bool,
+    pressure: float,
+) -> None:
+    """
+    Set minimize_kwargs dictionary values.
+
+    Parameters
+    ----------
+    minimize_kwargs : dict[str, Any]
+        Other keyword arguments to pass to geometry optimizer.
+    traj : Optional[str]
+        Path if saving optimization frames.
+    vectors_only : bool
+        Whether to optimize cell vectors, as well as atomic positions, by setting
+        `hydrostatic_strain` in the filter function.
+    pressure : float
+        Scalar pressure when optimizing cell geometry, in bar. Passed to the filter
+        function if either `vectors_only` or `fully_opt` is True.
```

Comment:
Does `fully_opt` exist here? is it part of `minimize_kwargs`?

Reply:
It's implicit at this point. `fully_opt` basically means the filter function isn't `None`, and `hydrostatic_strain` is `False`